crates/bevy_ecs/src/schedule/state.rs

```diff
@@ -37,6 +37,8 @@ pub use bevy_ecs_macros::States;
 ///
 /// ```
 pub trait States: 'static + Send + Sync + Clone + PartialEq + Eq + Hash + Debug + Default {
+    /// The type returned when iterating over all instances of this type.
```

Comment:
This is not iterating over instances but over variants.

Reply:
I mean it's a different way of saying the same thing. I didn't feel the need to repeat the word variant since it's already linked.